Makefile

```diff
@@ -50,20 +49,20 @@ eks-a-binary:
 
 .PHONY: eks-a-embed-config
 eks-a-embed-config: ## Build a dev release version of eks-a with embed cluster spec config
-	$(MAKE) eks-a-binary GIT_VERSION=$(DEV_GIT_VERSION) RELEASE_MANIFEST_URL=embed:///config/releases.yaml BUILD_TAGS='$(BUILD_TAGS) spec_embed_config'
```

Comment:
This won't wortk
The gitversion we use has to be present in the release manifest
And the dev release manifest won't have a release for GIT_VERSION